client/command.go

```diff
@@ -68,9 +69,11 @@ var clientGetKeyMetrics = func(map[string]string) {}
 func Run(
 	client knox.APIClient,
```

Comment:
instead of passing cache status in via the VisibilityParams -- create a new implementation of the `knox.APIClient` interface that does not use the cache. 

Callers can then use that new implementation and pass that directly in to the Run function -- and we will be certain the cache will not be used. I think as is, this change has gaps (like when fetching ACL -- it will still look in the cache).

Reply:
Add an uncached implementation of the knox.APIClient , but it has to produce some redundant code because golang does not support class inherited and function overriding.